src/python/pants/option/subsystem.py

```diff
@@ -135,7 +135,7 @@ def _is_default(self, __name: str) -> bool:
             assert isinstance(v, OptionsInfo)
 
             return (
-                self.options.is_default(__name)
+                self.options.is_default(__name.lstrip("_"))
```

Comment:
Interesting. What was the issue here?

Reply:
The class attribute is called `_go_search_paths`. `SubSystem.EnvironmentAware._is_default` inspects the class attribute on line 134, while `OptionsValueContainer` seems to contain the key `go_search_paths`, not the underscored prefixed-name. I assume the stripped variant is to make options?

Reply:
You're right: we're storing the raw value in `_go_search_paths` and exposing the post-processed value `go_search_paths` via a `@property`.